### PR TITLE
[Fix] Prevent divide-by-zero error on Ascend device for bbox_overlaps

### PR DESCRIPTION
**Pull request by dflhw**
Change the value of offset to adapt to the Ascend device.

### CODE REVIEW

File: mmcv/ops/csrc/pytorch/npu/bbox_overlaps_npu.cpp [hunk 1]
```diff
@@ -22,7 +26,7 @@ void bbox_overlaps_npu(const Tensor bboxes1, const Tensor bboxes2, Tensor ious,
       .Input(gtboxes)
       .Output(ious)
       .Attr("mode", modeStr)
-      .Attr("eps", (float)offset)
+      .Attr("eps", offset_)
```
Comment (zhouzaida) [inline mmcv/ops/csrc/pytorch/npu/bbox_overlaps_npu.cpp:29]: Hi, why is `eps (offset)` used as the denominator?

Comment (zhouzaida) [reply mmcv/ops/csrc/pytorch/npu/bbox_overlaps_npu.cpp:29]: Why is it set as `0.01` rather than other value?

Comment (dflhw) [reply mmcv/ops/csrc/pytorch/npu/bbox_overlaps_npu.cpp:29]: This is due to the limitation of the development mode and model scenario. 0.01 and 1 need to be enumerated.

**Review by grimoire — APPROVED** [review]
LGTM

**Pull request merged**